bibtex.html

```diff
@@ -0,0 +1,391 @@
+<!DOCTYPE html>
```

Comment:
Probably we should remove the old `index.html` and replace it by this, I can do that after merging.